src/client/cmd/list.h

```diff
@@ -24,6 +24,8 @@
 
 namespace multipass
 {
+class Formatter;
```

Comment:
Needed?

Reply:
We do since we have a `Formatter*` member and we didn't include the actual header for the `Formatter` class.

Reply:
Ok I see now, ta